src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -183,6 +184,22 @@ bool mongoConnect(const char* host,
 
     LM_I(("Successful connection to database"));
 
+    //
+    // WriteConcern
+    //
+    enum WriteConcern wconcern = (writeConcern == "normal")? W_NORMAL : W_NONE;
```

Comment:
Insted of "normal"/"none" (as string) I'd suggest to use 1 and 0 (as integer) directly. It would be clearer for Mongo DBAs.